### PR TITLE
Renaming2

### PR DESCRIPTION
**Pull request by jmprieur**
## Purpose
- More renamings
- Adding obsolete attributes to help customers move to the renamed methods. Which addresses https://github.com/Azure-Samples/active-directory-aspnetcore-webapp-openidconnect-v2/issues/231

## Does this introduce a breaking change?

```
[ ] Yes
[ X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ x] Refactoring (no functional changes, no api changes)
[ x] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
```

### CODE REVIEW

File: Microsoft.Identity.Web/WebApiServiceCollectionExtensions.cs [hunk 1]
```diff
@@ -23,6 +23,8 @@ namespace Microsoft.Identity.Web
     /// </summary>
     public static class WebApiServiceCollectionExtensions
     {
+        #region Compatibility
+        #endregion
         /// <summary>
```
Comment (jennyf19) [inline Microsoft.Identity.Web/WebApiServiceCollectionExtensions.cs]: move #endregion to the end?

File: Microsoft.Identity.Web/TokenCacheProviders/MsalAbstractTokenCacheProvider.cs [hunk 2]
```diff
@@ -110,6 +110,8 @@ public async Task ClearAsync()
         {
             // This is here a user token cache
             await RemoveKeyAsync(GetCacheKey(false)).ConfigureAwait(false);
+
+            // TODO: Clear the cookie session if any
```
Comment (jennyf19) [inline Microsoft.Identity.Web/TokenCacheProviders/MsalAbstractTokenCacheProvider.cs]: nit: maybe include the issue #w/this?

File: Microsoft.Identity.Web/WebApiServiceCollectionExtensions.cs (hunk 1 above)
Comment (jmprieur) [reply Microsoft.Identity.Web/WebApiServiceCollectionExtensions.cs]: Thanks addressed.

File: Microsoft.Identity.Web/TokenCacheProviders/MsalAbstractTokenCacheProvider.cs (hunk 2 above)
Comment (jmprieur) [reply Microsoft.Identity.Web/TokenCacheProviders/MsalAbstractTokenCacheProvider.cs]: Done. Thanks @jennyf19

**Pull request merged**